### PR TITLE
Add class b static reporting

### PR DESCRIPTION
**Pull request by sbender9**
Add static reports so that name, dimensions, callsign, etc. are available

### CODE REVIEW

**Comment by tkurki** [comment]
Hey this the first time that this stuff in the config file get used!

File: index.js [hunk 1]
```diff
@@ -14,29 +14,46 @@
...
       unsubscribe = Bacon.combineWith(function(position, sog, cog, head) {
         return createPositionReportMessage(mmsi, position.latitude, position.longitude, mpsToKn(sog), cog, head)
       }, ['navigation.position', 'navigation.speedOverGround', 'navigation.courseOverGroundTrue', 'navigation.headingTrue'].map(app.streambundle.getSelfStream, app.streambundle)).changes().debounceImmediate((props.updaterate || 60)*1000).onValue(msg => {
 
-        sendPositionReportMsg(msg, props.ipaddress, props.port)
+        sendReportMsg(msg, props.ipaddress, props.port)
       })
+
+      var static = getStaticInfo()
+      if ( Object.keys(static).length )
```
Comment (tkurki) [inline index.js]: When/if we add a ui for configuring the static data this plugin will pick up the change only after server restart. Would it hurt if this was checked every staticinterval anyway?

File: index.js [hunk 2]
```diff
@@ -14,29 +14,42 @@
...
       unsubscribe = Bacon.combineWith(function(position, sog, cog, head) {
         return createPositionReportMessage(mmsi, position.latitude, position.longitude, mpsToKn(sog), cog, head)
       }, ['navigation.position', 'navigation.speedOverGround', 'navigation.courseOverGroundTrue', 'navigation.headingTrue'].map(app.streambundle.getSelfStream, app.streambundle)).changes().debounceImmediate((props.updaterate || 60)*1000).onValue(msg => {
 
-        sendPositionReportMsg(msg, props.ipaddress, props.port)
+        sendReportMsg(msg, props.ipaddress, props.port)
       })
+
+      timeout = setInterval(function() {
+        info = getStaticInfo()
```
Comment (tkurki) [inline index.js]: Sorry, I should have been clearer: I thought that the condition was ok, but that it should be done periodically, and not just once and give up forever.

**Comment by sbender9** [comment]
I think the last commit is the best. Maybe I should redo this with just one commit?

**Pull request merged**